tools/nl/embeddings/run.sh

```diff
@@ -26,12 +26,16 @@ if [[ $# -le 1 ]]; then
   exit 1
 fi
 
-while getopts bflc OPTION; do
+while getopts beflc OPTION; do
   case $OPTION in
     b)
         echo -e "### Using the base default sentence_transformer model"
         FINETUNED_MODEL=""
         ;;
+    e)
+        MODEL_ENDPOINT_ID="$3"
```

Comment:
maybe add an example to the README using this option & include some information about what it does?

Reply:
can you also mention it in the README? I personally usually use the README to understand what a tool does instead of running help ..

Reply:
Added